src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -254,6 +260,107 @@ public void SarifTransformerTests_ToCurrent_OneRunWithFiles()
+                        ""semanticVersion"": ""2.1.0""
+                      },
+                      ""results"": [
+                      ]
+                    }
+                  ]
+                }";
+
+            SarifLog v2Log = TransformVersionOneToCurrent(v1LogText);
+
+            v2Log.Runs.Should().NotBeNull();
+            v2Log.Runs.Count.Should().Be(1);
+            v2Log.Runs[0].Resources.Should().NotBeNull();
+            v2Log.Runs[0].Resources.Rules.Should().NotBeNull();
+            v2Log.Runs[0].Resources.Rules.Count.Should().Be(3);
+
+            Rule rule = v2Log.Runs[0].Resources.Rules["C2001"];
+
+            rule.Id.Should().Be("C2001");
+            rule.Name.Should().BeNull();
+            rule.ShortDescription.Should().NotBeNull();
+            rule.ShortDescription.Text.Should().Be("A variable was used without being initialized.");
+            rule.FullDescription.Should().BeNull();
+            rule.Configuration.Should().BeNull();
+            rule.HelpLocation.Should().BeNull();
+            rule.MessageStrings.Should().NotBeNull();
+            rule.MessageStrings.Count.Should().Be(1);
+            rule.MessageStrings["default"].Should().Be("Variable \"{0}\" was used without being initialized.");
+            rule.Properties.Should().NotBeNull();
+            rule.Properties["some_key"].SerializedValue.Should().Be("\"FoxForceFive\"");
+
+            rule = v2Log.Runs[0].Resources.Rules["C2002"];
+
+            rule.Id.Should().Be("C2002");
+            rule.Name.Should().BeNull();
+            rule.ShortDescription.Should().BeNull();
+            rule.FullDescription.Should().NotBeNull();
+            rule.FullDescription.Text.Should().Be("Catfish season continuous hen lamb include dose copy grant.");
+            rule.Configuration.Should().NotBeNull();
+            rule.Configuration.Enabled.Should().Be(true);
+            rule.Configuration.DefaultLevel.Should().Be(RuleConfigurationDefaultLevel.Error);
+            rule.HelpLocation.Should().NotBeNull();
+            rule.HelpLocation.Uri.ToString().Should().Be("http://www.domain.com/rules/c2002.html");
+            rule.MessageStrings.Should().BeNull();
+            rule.Properties.Should().BeNull();
+
+            rule = v2Log.Runs[0].Resources.Rules["C2003"];
+
+            rule.Id.Should().Be("C2003");
+            rule.Name.Should().NotBeNull();
+            rule.Name.Text.Should().Be("Rule C2003");
+            rule.ShortDescription.Should().NotBeNull();
+            rule.ShortDescription.Text.Should().Be("Rules were meant to be broken.");
+            rule.FullDescription.Should().NotBeNull();
+            rule.FullDescription.Text.Should().Be("Rent internal rebellion competence biography photograph.");
+            rule.Configuration.Should().NotBeNull();
+            rule.Configuration.Enabled.Should().Be(false);
+            rule.Configuration.DefaultLevel.Should().Be(RuleConfigurationDefaultLevel.Warning);
```

Comment:
`"pass"` should map to `RuleConfigurationDefaultLevel.Note`.

Reply:
Fixed